### ISSUE TITLE
Allow portal.properties to be overridden by environmental variables

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#20068]
Many times, containerized customers want to make changes to the portal.properties that ship with dotCMS via environmental variables, just like we do with the properties in the dotmarketing-config.properties.  We need to allow customers to change these system properties as their installs dictates.

See support ticket: #103749

**Comment by nollymar** [comment]
Passed internal QA

**Comment by bryanboza** [comment]
@nollymar can you provide me more information how to test?

**Comment by nollymar** [comment]
I followed these steps: 
1) Set the property `auth.max.failures.limit` with a value different to 5 (as 5 is the default one) in the dotmarketing-config.properties
2) Modify the method `onFailureByEmailAddress` of the class `LoginMaxFailures.java`: 
```
public void onFailureByEmailAddress(String companyId, String emailAddress)
		throws AuthException {
        Logger.warn(this, "MAX FAILURE REACHED");
	}
```

This change will log "MAX FAILURE REACHED" when the max attempt defined in `auth.max.failures.limit` is reached.

In general, the idea is to set in the dotmarketing-config.properties file any property defined in the portal.properties. The value defined in the dotmarketing-config.properties should take precedence.

**Comment by bryanboza** [comment]
Fixed, tested on release 21.04 // Postgres // FF

**Comment by swicken-dotcms** [comment]
Reverted the changes in 5.2.8.5 as it was causing it to not load the config properly, creating issues in the backend.
